### DISCUSSION TITLE
Anomaly Transformer multidimensional

### DISCUSSION THREAD

**Discussion by lcoandrade** [sintel-dev/Orion#529] [Q&A] [answered]
Is it possible to run Anomaly Transformer with multidimensional data as in the original code?

**Comment by sarahmish** [comment, marked as answer]
Hi @lcoandrade, the pipeline in Orion currently only supports univariate data. If you'd like it to support multivariate input, please open an issue to request this feature!

Note that Orion only supports univariate output pipelines
